releasenotes/notes/enable-clustercheck-advanced-dispatching-edb4ed5f834c2ac6.yaml

```diff
@@ -0,0 +1,14 @@
+---
+enhancement:
+  - |
+    The Cluster Agent now enables both `CLUSTER_CHECKS_ADVANCED_DISPATCHING_ENABLED`
+    and `CLUSTER_CHECKS_REBALANCE_WITH_UTILIZATION` by default.
```

Comment:
```suggestion
    The Cluster Agent now enables both `DD_CLUSTER_CHECKS_ADVANCED_DISPATCHING_ENABLED`
    and `DD_CLUSTER_CHECKS_REBALANCE_WITH_UTILIZATION` by default.
```

Reply:
Looks like the variable names are prepended with `DD_` in the YAML file, but let me know if there's any reason not to include that here